### PR TITLE
Filters in column menu #3490

### PR DESCRIPTION
**Pull request by mnavasiolava**

### CODE REVIEW

File: core/src/main/web/outputdisplay/bko-tabledisplay/bko-tabledisplay.js [hunk 1]
```diff
@@ -77,6 +77,39 @@
...
+          var numValue = parseFloat(cellValue);
+          searchString = searchString.replace(/=/g, '==');
+          searchString = searchString.replace(/&&/g, '&& numValue');
+
+          try {
+            match = eval("numValue" + searchString);
+            if (!match) {
+              break;
+            }
+          } catch (e) {
```
Comment (tomca32) [inline core/src/main/web/outputdisplay/bko-tabledisplay/bko-tabledisplay.js:102]: We don't want to do anything with this error? Not even log it?

Comment (mnavasiolava) [inline core/src/main/web/outputdisplay/bko-tabledisplay/bko-tabledisplay.js:102]: this error occurs often enough, e.g. user start typing in filter field ">" or "> 0 && <", all these expressions cannot be evaluated and we'll get an error. idk if it makes sense to log it.

**Comment by scottdraves** [comment]
this is great.  a few glitches:
1) when i drag a column to reorder it, there are a bunch of extra hamburger menus in it.
2) the keyboard bindings don't work over the header of the column if another element has had the focus (try clicking on a code cell, then hover mouse over column header vs body cell).

**Comment by scottdraves** [comment]
i can't reproduce the problem with extra hamburger menus.  testing more...

**Comment by scottdraves** [comment]
:+1:

**Pull request merged**